build_tools/bin/symlink.js

```diff
@@ -21,11 +23,11 @@ async function symlink(module, name, source) {
     const current = process.cwd();
     try {
         const nodeModules = path.join(module, 'node_modules');
-        if (!(await exists(nodeModules))) {
+        if (!(await stat(nodeModules))) {
```

Comment:
Not sure this is the best fix, as I didn't kick the tires much on this and didn't dig into the affects on the `promisify` wrapper above

Reply:
I'm not sure if that is best either.

How about:
```
const exists = (s) => new Promise(r => fs.access(s, fs.F_OK, e => r(!e)))
```